src/mlpack/methods/ann/layer/CMakeLists.txt

```diff
@@ -59,6 +59,8 @@ set(SOURCES
   mean_pooling_impl.hpp
   multiply_constant.hpp
   multiply_constant_impl.hpp
+  multiply_merge.hpp
```

Comment:
Can you add the layer to the list of layer types in: https://github.com/mlpack/mlpack/blob/master/src/mlpack/methods/ann/layer/layer_types.hpp